bigchaindb/backend/mongodb/connection.py

```diff
@@ -12,12 +12,17 @@
                                            AuthenticationError)
 from bigchaindb.backend.connection import Connection
 
+from ssl import CERT_REQUIRED
```

Comment:
Lets move this to the beginning of the file. The way we try to organize the imports is:
```python
# standard lib
import time
import logging
from ssl import CERT_REQUIRED

# external dependencies
import pymongo

# internal dependencies
import bigchaindb
from bigchaindb.utils import Lazy
from bigchaindb.common.exceptions import ConfigurationError
from bigchaindb.backend.exceptions import (DuplicateKeyError,
                                           OperationError,
                                           ConnectionError,
                                           AuthenticationError)
from bigchaindb.backend.connection import Connection
```